README.md

```diff
@@ -103,6 +103,10 @@ When in SSH:
     cd /vagrant
     python3 chaos.py
 
+## Nbody simulation status
+
+![So chaotic](data/imgs/nbody.png)
```

Comment:
@PlasmaPower You are absolutely right, but I'm afraid I have no idea of how to do that. Should I close the PR?